relay-server/src/actors/project.rs

```diff
@@ -725,46 +732,24 @@ impl Project {
         })
     }
 
-    fn check_envelope_scoped(&mut self, message: CheckEnvelope) -> CheckEnvelopeResponse {
+    pub fn check_envelope_scoped(&mut self, message: CheckEnvelope) -> CheckEnvelopeResponse {
         let scoping = self.scope_request(message.envelope.meta());
         let result = self.check_envelope(message.envelope, &scoping);
         CheckEnvelopeResponse { result, scoping }
     }
 }
 
-impl Actor for Project {
-    type Context = Context<Self>;
-
-    fn started(&mut self, _ctx: &mut Self::Context) {
-        relay_log::debug!("project {} initialized without state", self.public_key);
-    }
-
-    fn stopped(&mut self, _ctx: &mut Self::Context) {
-        relay_log::debug!("project {} removed from cache", self.public_key);
-    }
-}
-
 /// Returns the project state if it is already cached.
 ///
 /// This is used for cases when we only want to perform operations that do
 /// not require waiting for network requests.
 #[derive(Debug)]
-pub struct GetCachedProjectState;
-
-impl Message for GetCachedProjectState {
-    type Result = Option<Arc<ProjectState>>;
+pub struct GetCachedProjectState {
```

Comment:
Since these are no longer messages on the project, they should also be moved to the file where they are implemented and handled.

Reply:
done